src/quick-lint-js/fe/parse.h

```diff
@@ -915,11 +916,21 @@ class parser {
 
 template <class ExpectedParenthesesError, class ExpectedParenthesisError,
           bool CheckForSketchyConditions, bool CheckForCommaOperator>
-void parser::parse_and_visit_parenthesized_expression(parse_visitor_base &v) {
+void parser::parse_and_visit_parenthesized_expression(
+    source_code_span token_span, parse_visitor_base &v) {
   bool have_expression_left_paren = this->peek().type == token_type::left_paren;
+  source_code_span left_paren_span = this->peek().span();
```

Comment:
This variable is misleading. The current token might not be a left_paren. (See the line above.)